Content.Shared/GameObjects/Components/Research/SharedMaterialStorageComponent.cs

```diff
@@ -29,8 +29,8 @@ public class SharedMaterialStorageComponent : Component, IEnumerable<KeyValuePai
         {
             get
             {
-                var ID = material.ID;
-                if (!Storage.ContainsKey(ID))
+                string ID = material.ID!;
+                if (!Storage.ContainsKey(ID!))
```

Comment:
should be handled.

Reply:
This complete indexer is unused, removed.